### ISSUE TITLE
Formatting crashes with `separate_kwargs_with_semicolon = true`

### ISSUE THREAD

**Issue by iago-lito** [fannheyward/coc-julia#351]
For some reason, all options I set in `.JuliaFormatter.toml` work except this one:

`.JuliaFormatter.toml`
```toml
separate_kwargs_with_semicolon = true
```

`main.jl`
```julia
function f(a, b = 5) end
```

`minimal.vimrc`
```nvim
set nocompatible
set termguicolors
set runtimepath^=$HOME/.vim/plugged/coc.nvim
```

To reproduce the crash:
```
$ nvim -u minimal.vimrc main.jl
:CocEnable
:call CocAction('format')
```

I get
```
[coc.nvim] Failed to format document: MethodError(Core.var"#Type##kw"(), ((indent = 4, margin = 92, always_for_in = false, whitespace_typedefs = false, whitespace_ops_in_indices = false, remov
e_extra_newlines = false, import_to_using = false, pipe_to_function_call = false, short_to_long_function_def = false, always_use_return = false, whitespace_in_kwargs = true, annotate_untyped_f
ields_with_any = true, format_docstrings = false, align_struct_field = false, align_assignment = false, align_conditional = false, align_pair_arrow = false, conditional_to_if = false, normaliz
e_line_endings = "auto", align_matrix = false, trailing_comma = true, separate_kwargs_with_semicolon = true), JuliaFormatter.Options), 0xffffffffffffffff). use :CocOpenLog for details
```
With pretty much the same message in `:CocOpenLog`.

Interestingly, formatting does work from a plain Julia REPL:
```julia
julia> using JuliaFormatter
julia> format("main.jl")
```
So I guess it's not a bug with `JuliaFormatter.jl`.

Also, it does work from Coc with the following config file instead:
`.JuliaFormatter.toml`
```toml
indent = 1
```
So I guess the bug is only related to the `separate_kwargs_with_semicolon` option?

**Comment by lassepe** [comment]
The option `separate_kwargs_with_semicolon` was added in `0.23.0` of `JuliaFormatter` ([link](https://github.com/domluna/JuliaFormatter.jl/commit/7ffeeb6d7e84676161b75d195057d3c2afeabb2f)) but the environment shipped with this extension has version `0.20.5`. ([link](https://github.com/fannheyward/coc-julia/blob/61e9cd225df54896f3d1c9775cd9ba48c87afebc/server/JuliaLS/Manifest.toml#L73)). So this would probably be fixed by updating the language server env here and making a new release. In the meantime, you could also do that manually by updating the Manifest in `~/.config/coc/extensions/node_modules/coc-julia/server/JuliaLS` like you would for a usual env (and potentially re-build your system image if you use that).

**Comment by fannheyward** [comment]
coc-julia 0.13.0 has upgrade JuliaFormatter to 1.0.10.
